### DISCUSSION TITLE
stop connection in link : wss://broker.emqx.io:8083/mqtt

### DISCUSSION THREAD

**Discussion by actionoise** [emqx/MQTTX#1847] [General]
good Morning , please would be possible to have information about the problem of connection from the link:
wss://broker.emqx.io:8083/mqtt i have problem to connection from yesterday , 
i also try to connect to mqttx server in "broker.emqx.io" from the app mqttx and i have the same problem. 
Thanks in advance for your help.

**Comment by ysfscream** [comment]
Hi, please try to use port `8084` to connect to the wss host.

**Reply by actionoise** [reply]
thanks for your advice, i used everitime the 8084 port , but yesterday the server was off with ws and port 8083 and wss and port 8084,
Today it started working again, thanks a lot for your service 
have a good day.

**Reply by ysfscream** [reply]
You're welcome. Welcome to https://www.emqx.com/en/cloud get your own service 😀

**Reply by actionoise** [reply]
Hello , i have same problem from today, in Port 8084 wss, connection failed.

**Reply by actionoise** [reply]
Ok , now working....!!!👍🏻
